platform/nodejs/test.js

```diff
@@ -134,7 +134,7 @@ async function doHNTrie() {
 }
 
 async function spawnMocha() {
-    await promisify(spawn)('mocha', [ 'tests' ], { stdio: [ 'inherit', 'inherit', 'inherit' ] });
+    await promisify(spawn)('mocha', [ '--experimental-vm-modules', '--no-warnings', 'tests' ], { stdio: [ 'inherit', 'inherit', 'inherit' ] });
```

Comment:
Without `--experimental-vm-modules` the `vm.Module` API is not available.

Without `--no-warnings` we get warnings about using this experimental API.

We do want to show other kinds of warnings, and for this we add a handler for the `warning` event, which is triggered despite `--no-warnings`.